### DISCUSSION TITLE
Feature request: add emoji icon "Done" to comment reactions.

### DISCUSSION THREAD

**Discussion by tetianaman** [community/community#114260] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

My friends used to mark those comments we've finished work about as 🚀 . But nowadays when Ukrainians, Israelis etc live under the rocket attacks, it's a cringe. No better icon to mean "done". Add please ✔️, ✅ or something similar. 

P.S. I know about label "Outdated" for a file. That's different things.

**Comment by ghost** [comment]
+1 - since the 👀 represents the concept of "looking into it", there should be a concept of "completed/done" as well. The "rocket ship" is the closest equivalent but a ✔️, ✅ would definitely be better

**Comment by benjub** [comment]
Indeed, the "Done" reaction emoji (probably a checkmark ✅) is really missing: its meaning is clear, and this would avoid cluttering threads with comments merely saying "Done". This request keeps coming up, see for instance #35992.
